src/sentry/workflow_engine/handlers/condition/event_attribute_handler.py

```diff
@@ -15,6 +15,16 @@
 class EventAttributeConditionHandler(DataConditionHandler[WorkflowJob]):
     type = DataConditionHandlerType.ACTION_FILTER
 
+    comparison_json_schema = {
+        "type": "object",
+        "properties": {
+            "attribute": {"type": "string"},
+            "match": {"type": "string", "enum": [*MatchType]},
+            "value": {"type": "string"},
+        },
+        "required": ["attribute", "match", "value"],
+    }
```

Comment:
can we also add `"additionalProperties": False` and test adding additional keys into `comparison` fails?